tests/test_clean.py

```diff
@@ -1,96 +1,210 @@
+import os
+
 from html5lib.filters.base import Filter
 import pytest
 
-import bleach
+from bleach import clean
```

Comment:
I changed all instances of `bleach.clean` to `clean` figuring there aren't other instances of "clean", so might as well go for some easy-win brevity.